docs/CONNECTOR_PROTOCOL.md

```diff
@@ -252,11 +256,21 @@ In addition to the connector index `.elastic-connectors`, we have an additional
     }
   };
   index_name: string; -> The name of the content index
-  worker_hostname: string; -> The hostname of the worker to run the job
   indexed_document_count: number; -> Number of documents indexed in the job
-  deleted_document_count: number; -> Number of documents deleted in the job
-  created_at: date; -> The date/time when the job is created
-  completed_at: date; -> The data/time when the job is completed
+  indexed_document_volume: number; -> The volume (in bytes) of documents indexed in the job
+  last_seen: date; -> Connector writes check-in date-time regularly (UTC)
+  metadata: object; -> Connector-specific metadata
+  pipeline: { -> Connector pipeline
```

Comment:
I wonder if you'd considered to put `pipeline` and other job's configurations option under `properties` object

Reply:
Lets not use `properties` just so it doesn't get confusing with Elasticsearch where `properties` is used to describe sub-object fields 😓 

To be fair to Chenhui, he'd argued in another thread to use `connector` as the outer key to group all this stuff under. I'd noted that Sander has expressed wanting to keep things flatter, and I (when I added `filtering`) didn't want to nest it by itself. Chenhui, feel free to blame this on me and say, "I told you so!" :D

Reply:
It won't bring any performance benefits, eventually, they will be flattened. But I'm still in favor of a grouped object, which makes it much easier to read, especially now all the fields are alphabetically ordered.